build.tmpl.yml

```
@@ -228,66 +225,70 @@ archives:
   - id: windows-cgo
     builds:
       - windows-cgo
-    replacements:
-      darwin: macOS
-      386: 32bit
-      amd64: 64bit
     format_overrides:
       - goos: windows
         format: zip
-    name_template: "{{ .ProjectName }}_{{ .Version }}-{{ .Os }}_sqlite_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
+    name_template: >-
+      {{ .ProjectName }}_{{ .Version }}-windows_sqlite_amd64
 
   - id: macos-cgo
     builds:
       - macos-cgo
-    replacements:
-      darwin: macOS
-      386: 32bit
-      amd64: 64bit
-    format_overrides:
-      - goos: windows
-        format: zip
-    name_template: "{{ .ProjectName }}_{{ .Version }}-{{ .Os }}_sqlite_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
+    name_template: >-
+      {{ .ProjectName }}_{{ .Version }}-macos_sqlite_{{ if .Arch eq "amd64" }}64bit{{ else }}{{ .Arch }}{{ end }}
```

Please make sure that the casing is correct, otherwise our install scripts will fail:

```suggestion
      {{ .ProjectName }}_{{ .Version }}-macOS_sqlite_{{ if .Arch eq "amd64" }}64bit{{ else }}{{ .Arch }}{{ end }}
```

Also it seems that `{{ if .Arm }}v{{ .Arm }}{{ end }}` is missing?

---

macOS only comes in amd64 or arm64, so I tried to simplify the template.